packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -320,22 +320,24 @@ Lambda triggers can either be specified as part of the `UserPool` initialization
 on the construct, as so -
 
 ```ts
-import * as lambda from '@aws-cdk/aws-lambda';
-
 const authChallengeFn = new lambda.Function(this, 'authChallengeFn', {
-  // ...
+  runtime: lambda.Runtime.NODEJS_10_X,
+  handler: 'index.handler',
+  code: lambda.Code.fromInline('auth challenge'),
 });
```

Comment:
Is this necessary? I thought rosetta translations work as long as it was valid typescript. Does it have to actually compile and work?

I would've preferred to leave this open for the user to define and rosetta translation snippet does the same.